doc/administrator/administrate.rst.tmpl

```diff
@@ -45,10 +45,18 @@ Each metadata entry has the following attributes:
 
 To set a metadata entry, create or edit an entry in the Metadata view of the administration UI.
 Regarding effect on the referenced tree item on the client side, you will find a description for each sort
-of metadata in the `NGEO documentation
-<https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/apidoc/gmfThemes.GmfMetaData.html>`_.
+of metadata in the ``GmfMetaData`` definition in `themes.js
+<https://github.com/camptocamp/ngeo/blob/${MAIN_BRANCH}/contribs/gmf/src/themes.js>`_.
+
+Some Metadata are used by the layers views (for editing) on the server side:
```

Comment:
Some metadata items are used by the layer views (for editing) on the server side: